ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -422,13 +432,14 @@ objects:
 
         cp "$(command -v openshift-install)" /tmp/shared
         mkdir ${ARTIFACT_DIR}/installer
+        mkdir -p "${XDG_RUNTIME_DIR}"
 
         if [[ -n "${INSTALL_INITIAL_RELEASE}" && -n "${RELEASE_IMAGE_INITIAL}" ]]; then
           echo "Installing from initial release ${RELEASE_IMAGE_INITIAL}"
           OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_INITIAL}"
         elif [[ "${CLUSTER_VARIANT}" =~ "mirror" ]]; then
           # mirror the release image and override the release image to point to the mirrored one
-          mkdir /tmp/.docker && cp /etc/openshift-installer/pull-secret /tmp/.docker/config.json
+          mkdir "${XDG_RUNTIME_DIR}/containers" && cp /etc/openshift-installer/pull-secret "${XDG_RUNTIME_DIR}/containers/auth.json"
```

Comment:
hm, this seems it might brake eg. `release-openshift-ocp-installer-e2e-aws-csi-4.9`. Need to test and we probably have to support the `~docker/config.json` for the old releases